protocol/avdl/notify_paperkey.avdl

```diff
@@ -0,0 +1,8 @@
+
+@namespace("keybase.1")
+protocol NotifyPaperKey {
+  import idl "common.avdl";
+
+  @notify("")
+  void paperKeyCached(UID uid, KID encKID, KID sigKID);
```

Comment:
It's likely the first time I've mentioned it.  It's a new feature that I added, since it's the way the AVDL spec (now?) says to do it.  It emits the same code, and just is slightly cleaner to look at.  I would say use it here and everyone going forward, and we can make a sweep to change over the existing `@notify("")`.